environment.yaml

```diff
@@ -0,0 +1,17 @@
+name: gene_dependency_representations
```

Comment:
please change the filename to `environment.yml`

(notice that there are two environment files being modified in this PR)